examples/js/loaders/GLTFLoader.js

```diff
@@ -615,29 +601,18 @@ THREE.GLTFLoader = ( function () {
 					'material.specularColor = specularFactor.rgb;',
 				].join( '\n' );
 
-				var fragmentShader = shader.fragmentShader
-					.replace( 'uniform float roughness;', 'uniform vec3 specular;' )
```

Comment:
this is a bit weird, it may not be the best practice to have these "dangling" uniforms in the shader templates,